Gruntfile.coffee

```diff
@@ -356,6 +356,14 @@ module.exports = (grunt) ->
         src: 'plugins/*'
         dest: '.tmp/js'
 
+      services:
```

Comment:
[minor] What if we rename this to `alias` or `skroutza`? 🤔

Reply:
good point, this was a leftover name to be honest